File: bitget-golang-sdk-api/pkg/model/mix/account/setmarginmodereq.go
package account

/**
 * @Author: bitget-sdk-team
 * @Date: 2022-09-30 10:46
 * @DES: Adjust margin mode request
 */
type SetMarginModeReq struct {
	/**
	 * Margin mode
	 */
	MarginMode string `json:"marginMode"`
	/**
	 * Currency pair
	 */
	Symbol string `json:"symbol"`
	/**
	 * Deposit currency
	 */
	MarginCoin string `json:"marginCoin"`
}

File: bitget-node-sdk-api/src/lib/model/mix/account/SetMarginModeReq.ts
export class SetMarginModeReq {
    /**
     * Deposit currency
     */
    private _marginCoin!:string;
    /**
     * Currency pair
     */
    private _symbol!:string;
    /**
     * Margin mode
     */
    private _marginMode!:string;

    get marginCoin(): string {
        return this._marginCoin;
    }

    set marginCoin(value: string) {
        this._marginCoin = value;
    }

    get symbol(): string {
        return this._symbol;
    }

    set symbol(value: string) {
        this._symbol = value;
    }

    get marginMode(): string {
        return this._marginMode;
    }

    set marginMode(value: string) {
        this._marginMode = value;
    }
}